phel

```diff
@@ -19,4 +20,9 @@ require $autoloadPath;
 Gacela::bootstrap($projectRootDir, Phel::configFn());
 
 $bootstrap = new ConsoleBootstrap();
-$bootstrap->run();
+$bootstrap->setAutoExit(false);
+$exitCode = $bootstrap->run();
+
+(new FilesystemFacade())->clearAll();
```

Comment:
Can we place the `(new FilesystemFacade())->clearAll();` line inside the `$bootstrap->run()`? 🤔

Reply:
Yes, good idea! I will do that